lib/openshift/cluster_logging.rb

```diff
@@ -0,0 +1,107 @@
+# represent a clusterlogging object
+module BushSlicer
+  class ClusterLogging < ClusterResource
+    RESOURCE = "clusterloggings"
+
+    ### represent status section ["collection", "curation", "logStore", "message", "visualization"]
+    private def collection_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'collection')
+    end
+    private def curation_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'curation')
+    end
+
+    private def log_store_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'logStore')
+    end
+
+    private def message_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'message')
+    end
+
+    private def visualization_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'visualization')
+    end
+
+
+    private def fluentd_status_raw(user: nil, cached: true, quiet: false)
+      collection_raw(user: user, cached: cached, quiet: quiet).dig('logs', 'fluentdStatus')
+    end
+
+    private def curator_status_raw(user: nil, cached: true, quiet: false)
+      self.curation(user: user, cached: cached, quiet: quiet)['curatorStatus']
+    end
+
+    def fluentd_pods(user: nil, cached: true, quiet: false)
+      fluentd_status_raw(user: user, cached: cached, quiet: quiet)['pods']
+    end
+
+    def fluentd_ready_pods(user: nil, cached: true, quiet: false)
+      self.fluentd_pods(user: user, cached: cached, quiet: quiet)['ready']
+    end
+
+    def fluentd_failed_pods(user: nil, cached: true, quiet: false)
+      self.fluentd_pods(user: user, cached: cached, quiet: quiet)['failed']
+    end
+
+    def fluentd_notready_pods(user: nil, cached: true, quiet: false)
+      self.fluentd_pods(user: user, cached: cached, quiet: quiet)['notReady']
+    end
+
+    private def es_status_raw(user: nil, cached: true, quiet: false)
+      log_store_raw(user: user, cached: cached, quiet: quiet)['elasticsearchStatus']
+    end
+
+    def es_cluster_health(user: nil, cached: true, quiet: false)
+      es_status_raw(user: user, cached: cached, quiet: quiet).first['clusterHealth']
+    end
+
+    def es_pods(user: nil, cached: true, quiet: false)
+      es_status_raw(user: user, cached: cached, quiet: quiet).first['pods']
+    end
+
+    def kibana_status(user: nil, cached: true, quiet: false)
+      visualization_raw(user: user, cached: cached, quiet: quiet)['kibanaStatus']
+    end
+
+    def kibana_pods(user: nil, cached: true, quiet: false)
+      self.kibana_status(user: user, cached: cached, quiet: quiet).first['pods']
+    end
```

Comment:
all these methods also return some structure instead of objects user can more easily interact with. And not used outside the class. So `es_cluster_health`, `es_pods`, `kibana_status` and `kibana_pods` can happily be private.